explorer/frontend/Dockerfile

```diff
@@ -6,16 +6,13 @@ FROM base AS deps
 RUN apk add --no-cache libc6-compat
 WORKDIR /app
 
-# Install dependencies based on the preferred package manager
-COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
+# Install dependencies using npm
+COPY package.json package-lock.json* ./
+RUN npm install
```

Comment:
Is the below line needed when npm install is here? 🤔

Reply:
Thanks for pointing that out. I missed it—there should only be one installation command here. I've removed the second one.